cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -784,6 +789,173 @@ private void checkCurrentRuns(int retries, String url, int currentRunningProgram
     Assert.assertTrue(output.size() == currentRunningProgramsExpected);
   }
 
+  protected int getWorkflowSuspendResume(String namespaceId, String appId, String workflowId, String runId,
```

Comment:
should we have ``suspendWorkflow(Id.Program, runId)`` and ``resumeWorkflow(Id.Program, runId)`` instead? @chtyim recently refactored this class similarly, so we should follow that pattern.